media/js/firefox/accounts.js

```diff
@@ -124,27 +122,81 @@
                 showFxAccountsForm();
             }
         }, 2500);
-
-        //
-        setTimeout(Mozilla.syncAnimation, 1000);
     }
 
-    // if user is not on Firefox for Desktop, redirect to the /firefox/sync/ page.
-    if (window.Mozilla.Client.isFirefoxDesktop) {
-        // if uitour callback does not fire in 500ms, load the iframe anyway.
-        _uitourTimeout = setTimeout(loadFxAccountsForm, 500);
-        // if user is signed into sync, redirect the /firefox/sync/ page,
-        // else load the FxA iframe.
+    function showFxAForm() {
+        loadFxAccountsForm();
+
+        // UITour is used only to show conditional messaging.
         Mozilla.UITour.getConfiguration('sync', function (config) {
-            clearTimeout(_uitourTimeout);
             if (config.setup) {
-                redirectToSyncPage();
+                // Show signed-in messaging if user is already using Sync
+                $('.signed-in').show();
             } else {
-                loadFxAccountsForm();
+                // Show sign-up steps if user is not already using Sync
+                $('.sign-up-instructions').show();
             }
         });
-    } else {
-        redirectToSyncPage();
     }
 
+    // Firefox iOS users get SUMO link
+    function showFirefoxiOS() {
+        $main.addClass('single-column');
+        $('.mobile').show();
+        $('.firefox-ios').css('display', 'inline-block');
+    }
+
+    // Firefox Android users get SUMO link
+    function showFirefoxAndroid() {
+        $main.addClass('single-column');
+        $('.mobile').show();
+        $('.firefox-android').css('display', 'inline-block');
```

Comment:
Not a blocker for this PR, but wondering if we should begin enforcing `display` changes with classes? Could be a nice technical debt task.